docs/source/release_notes.rst

```diff
@@ -2,14 +2,16 @@
 
 Release Notes
 -------------
-.. **Future Release**
+**Future Release**
     * Enhancements
     * Fixes
     * Changes
     * Documentation Changes
     * Testing Changes
+        * Add ``pyspark`` and ``koalas`` to automated dependency checks (:pr:`1192`)
```

Comment:
Wrong comment?

Reply:
Left an actual change log line.  Maybe when we add koalas 1.3.0 support back in, we can delete this log entry and add this pr number to the new log entry.